kalamari/kalamari.py

```diff
@@ -16,14 +17,30 @@ def __init__(self, json):
                 self.json = Tree.tree_from_dict(loads(json))
 
     def __repr__(self):
-        result = {}
-        for node in self.json[1]:
-            if node.children:
-                result[node.data] = "{...}"
-            if node.container:
-                result[node.data] = "..."
+        try:
+            result = {}
+            for node in self.json[1]:
```

Comment:
When the tree is empty, especially from loads('""') or loads('{}'), none of the tree accesses to `[1]` will work as intended.

Reply:
Hey @rjauquet , thanks for putting effort into this. Good catch!